lib/Yam/Agama/Pom/GrubMenuSLESPage.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
```

Comment:
lower letter for all the acronysm please, 'sles'.

Reply:
Renamed to `GrubMenuSlesPage`